backend/app/Controllers/Admin/NodeStatusController.php

```
@@ -100,7 +101,8 @@ public function getGlobalStatus(Request $request): Response
                     $node['daemonListen'],
                     $node['scheme'],
                     $node['daemon_token'],
-                    10 // Short timeout for status checks
+                    10 // Short timeout for status checks,
+                    WingsUrlHelper::isBehindProxy($node)
```

PHPStan (2.2.5)

[error] Syntax error, unexpected T_STRING, expecting ')' on line 105


(phpstan.parse)